core/src/test/scala/unit/kafka/coordinator/transaction/TransactionCoordinatorTest.scala

```diff
@@ -495,10 +495,62 @@ class TransactionCoordinatorTest {
 
     EasyMock.replay(transactionManager)
 
-    coordinator.handleEndTransaction(transactionalId, 0, 0, TransactionResult.COMMIT, errorsCallback)
+    coordinator.handleEndTransaction(transactionalId, 0, 0, TransactionResult.COMMIT, false, errorsCallback)
     assertEquals(Errors.COORDINATOR_LOAD_IN_PROGRESS, error)
   }
 
+  @Test
+  def shouldReturnInvalidEpochWhenEndTxnIsFromClientAndEpochIsLarger(): Unit = {
+    val serverProducerEpoch = 1.toShort
```

Comment:
At a quick glance, these tests look basically the same except for the epoch which is sent and the expected error. Is it possible to factor out a helper?